Sources/Purchasing/Purchases.swift

```diff
@@ -1829,6 +1852,14 @@ extension Purchases: PurchasesOrchestratorDelegate {
         self.delegate?.purchases?(self, readyForPromotedProduct: product, purchase: startPurchase)
     }
 
+    #if os(iOS) || targetEnvironment(macCatalyst)
+    @objc
+    @available(iOS 13.4, macCatalyst 13.4, *)
+    internal func shouldShowPriceConsent() -> Bool {
```

Comment:
internal here is redundant

Reply:
if it's internal, does it have to be `@objc`?

Reply:
I'd ask the same two questions about the existing method, maybe it's a leftover from the migration?

Reply:
Yeah, that's right. This is copypasta.